build.gradle

```diff
@@ -519,12 +520,32 @@ forbiddenApisTest.enabled = false
  * Maven Publish *
  *****************/
 
+void addManifest(Jar jar) {
+    jar.manifest {
+        attributes 'Implementation-Title': project.name
+        attributes 'Implementation-Vendor': 'HiveMQ'
+        attributes 'Implementation-Version': project.version
+    }
+}
+
+task embeddedJar(type: Jar) {
```

Comment:
why is this needed, we already have a jar task that does the same

Reply:
you mean jar?

Reply:
yes, jar packages only our classes (shadowJar packages also all dependencies)